### PR TITLE
:hammer: apply trampss-redux-factory on gui model reducer

### PR DESCRIPTION
**Pull request by bpetetot**

### CODE REVIEW

**Comment by fabienjuif** [comment]
I put the `not-ready` label.
Feel free to remove it if you think this is a mistake :)

File: src/gui/redux/model/model.spec.js [hunk 1]
```diff
@@ -1,91 +1,4 @@
...
-      const state = reducer(prevState, updateProp('foo', 'string', 'bar'))
-
-      // asserts
-      expect(state).toBeDefined()
-      expect(state.foo).toBeDefined()
-      expect(typeof state.foo).toBe('string')
-      expect(state.foo).toEqual('bar')
-    })
-  })
+  it.skip('should initialize state')
```
Comment (fabienjuif) [inline src/gui/redux/model/model.spec.js]: Oups :)
Maybe test middlewares at least ?

File: src/gui/redux/model/model.selectors.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+import model from './model'
 import { convertToGuiValue } from '../utils/docgen'
 
-const getModel = state => state.model
+const getComponentValue = (name, type) => state =>
+  convertToGuiValue(model.get(name)(state).value, type)
 
-const getComponentValue = (name, type) =>
-  createSelector(getModel, model => convertToGuiValue(model[name], type))
+const getComponentProps = state =>
```
Comment (fabienjuif) [inline src/gui/redux/model/model.selectors.js]: These selectors are not optimized, reselect ?

**Comment by bpetetot** [comment]
👌 done @fabienjuif

**Review by fabienjuif — APPROVED** [review]
👌

**Pull request merged**